.ci/ansible/playbook.yml

```
@@ -93,11 +93,16 @@
       vars:
         golang_version: 1.17
   vars:
-    ansible_user: admin
     ansible_python_interpreter: /usr/bin/python3
   pre_tasks:
     - apt: {update_cache: yes}
   tasks:
+  - include_vars: '{{ ansible_distribution }}.yml'
+
+  - name: Print Ansible User obtained from the distribution vars
+    ansible.builtin.debug:
+    var: "ansible_user"
```

What do you think this?

```suggestion
  - name: Print Ansible User obtained from the distribution vars
    ansible.builtin.debug:
      msg: ansible_user {{ ansible_user }}
```

or

```suggestion
  - name: Print Ansible User obtained from the distribution vars
    ansible.builtin.debug:
      var: ansible_user
```

from https://docs.ansible.com/ansible/latest/collections/ansible/builtin/debug_module.html#examples